ds_judgements_public_ui/templates/pages/home.html

```diff
@@ -13,6 +13,6 @@
 
 {% block content %}
   {% include "includes/basic_search_form.html" %}
+  {% include "includes/cards_homepage.html" %}
   {% include "includes/recent_judgments.html" %}
-  {% include "includes/browse_by_court.html" %}
```

Comment:
I'm worried about this loss of homepage functionality; I thought these were quite heavily used, and now they're effectively hidden behind the "browse courts" link (which is to a very lengthy page).